File: js/PROBLEMS/1114.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(function(item){
    return item
})

for (i=0;i<lines.length;i++){
    x=lines[i][0]
    y=lines[i][1]

    if (lines[i] !== '2002'){
        console.log('Senha Invalida')
    }else{
        console.log("Acesso Permitido")
        break;
    }
}

File: c/PROBLEMS/1114.c
#include <stdio.h>

int main(){
    while (1){
        int e;
        scanf("%d", &e);
        if (e!=2002){
            printf("Senha Invalida\n");
        }else{
            printf("Acesso Permitido\n");
            break;
        }
    }
}